src/utils/blockchain/contracts.testnet.ts

```diff
@@ -59,6 +59,10 @@ export const contracts = {
     abi: abiTestWBRTCToken,
     blockNumber: 1205599,
   },
+  RBTC_itoken: {
+    address: '0xe67Fe227e0504e8e96A34C3594795756dC26e14B',
+    abi: abiTestWBRTCToken as any,
+  },
```

Comment:
Remove `itoken`s and use `_lending` instead - it's should be same contracts and already added to the codebase

Reply:
done